Library/Account.cs

```diff
@@ -510,7 +511,21 @@ public static void Reopen(string accountCode)
         /// <returns></returns>
         public static RecurlyList<Account> List(Account.AccountState state = Account.AccountState.Active)
         {
-            return new AccountList(Account.UrlPrefix + "?state=" + state.ToString().EnumNameToTransportCase());
+            return List(state, null);
+        }
+
+        /// <summary>
+        /// Lists accounts, limited to state
+        /// </summary>
+        /// <param name="state">Account state to retrieve</param>
+        /// <param name="sort">SortParams used to describe server sorting</param>
+        /// <returns></returns>
+        public static RecurlyList<Account> List(Account.AccountState state, SortParams sort)
+        {
+            sort = sort.Equals(null) ? new SortParams() : sort;
+            var parameters = sort.toNamedValueCollection();
+            parameters["state"] = state.ToString().EnumNameToTransportCase();
+            return new AccountList(Account.UrlPrefix + "?" + parameters.ToString());
```

Comment:
I may also add a constructor to the `RecurlyList` abstract class to take a base url and a `NamedValueCollection` that way we don't have to keep writing this code.